src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -354,6 +354,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({
             <NavBarSchedule
               isVisible={isEventScheduleVisible}
               venueId={venueId}
+              closeEventSchedule={() => toggleEventSchedule()}
```

Comment:
Just `closeEventSchedule={toggleEventSchedule}` that way the JS won't re-create the reference on every single rerender.

Reply:
fixed in 5afa7cde5b6bd9d643aeb30b9b72287e5fc9bb5b